### PR TITLE
PMM-9752 integrate pmm-dump

### PR DESCRIPTION
**Pull request by atymchuk**
Add pmm-dump to `/usr/local/bin`

- [ ] https://github.com/Percona-Lab/pmm-submodules/pull/2404

### CODE REVIEW

File: ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Install pmm-dump, always overwrite the version before with latest
+
+- name: Download pmm-dump binary (tech preview)
+  get_url:
+    url: https://github.com/percona/pmm-dump/releases/download/0.5.1-beta/pmm-transferer
+    dest: /tmp/pmm-transferer
+    tmp_dest: /tmp/
+    force: yes
+    headers:
+      Accept: "application/octet-stream"
```
Comment (nikita-b) [inline ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml:11]: It won't work without internet :(

File: ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- name: Display the File contents
+  shell: ls -laH /tmp/
```
Comment (nikita-b) [inline ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml:14]: it's better to use `command` module instead `shell` when it's possible.

File: ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+# Install pmm-dump, always overwrite the version before with latest
+
+- name: Download pmm-dump binary (tech preview)
+  get_url:
+    url: https://github.com/percona/pmm-dump/releases/download/0.5.1-beta/pmm-transferer
+    dest: /tmp/pmm-transferer
```
Comment (nikita-b) [inline ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml:7]: Why we can't save file right to `/usr/local/bin/pmm-dump`?

**Review by nikita-b — CHANGES_REQUESTED** [review]

File: ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml (hunk 2 above)
Comment (atymchuk) [reply ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml:14]: Thanks! It's to debug, I'll remove it anyway.

File: ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml (hunk 3 above)
Comment (atymchuk) [reply ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml:7]: Sure, will do

File: ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml (hunk 1 above)
Comment (atymchuk) [reply ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml:11]: Should we just put the file at build time, not download it?

Comment (atymchuk) [reply ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml]: I removed the playbook in favor of a separate spec.

**Review by nikita-b — APPROVED** [review]

**Review by denisok — APPROVED** [review]

**Pull request merged**